requirements-testing.txt

```diff
@@ -5,7 +5,6 @@ pytest==5.3.2
 pytest-cov==2.8.1
 autopep8==1.4.4
 mypy==0.761
-py-spy==0.3.0
```

Comment:
FYI, if we don't drop this, the pip install -r will not work : 
`Could not find a version that satisfies the requirement py-spy==0.3.0`